frontend/src/components/admin/companies/company-card/index.tsx

```diff
@@ -24,7 +25,9 @@ const CompanyCard: FC<Props> = ({ company }) => {
               <Card.Img src="holder.js/100px180" alt={company.name} />
               <Card.Title className="fs-1">{company.name}</Card.Title>
             </div>
-            <Button onClick={handleShow}>edit</Button>
+            <Button themeType={''} onSubmit={handleShow}>
```

Comment:
you have default themeType value in Button.tsx so themeType={''} should be removed